src/jrd/trace/TraceConfigStorage.cpp

```diff
@@ -470,15 +470,20 @@ void ConfigStorage::compact()
 	ULONG check_used, check_size;
 	check_used = check_size = sizeof(TraceCSHeader);
 
+	// Track undeleted slots from dead storages
+	Firebird::NonPooledMap<ULONG, bool> deadStorages;
```

Comment:
The second field (bool) is not used, why not use `SortedArray` or `SparseBitmap` here ?
I would use `SortedArray` with `InlineStorage` as there should not be too many items.
Also, this container used to collect PID's of dead processes, not storages.
Names are important, please use something more appropriate.

Reply:
I didn't find a Firebird::set class in the code, so I used a map. If SortedArray is OK, then I use it